benchmarks/inc/udt.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+template <typename Contained>
+struct trivial {
+    Contained c;
+};
+
+template <typename Contained>
+inline bool operator==(const trivial<Contained>& lhs, const trivial<Contained>& rhs) {

```

Comment:
We conventionally don't mark function templates as `inline` (as function templates already get the partial ODR exemption, and we don't care about the inlining hint). Occurs below.